spotbugs/src/main/java/edu/umd/cs/findbugs/ba/jsr305/TypeQualifierResolver.java

```diff
@@ -68,7 +68,6 @@ public class TypeQualifierResolver {
     };
 
     private static final ClassDescriptor[] NULLABLE_CLASS_DESCRIPTORS = new ClassDescriptor[] {
-        DescriptorFactory.createClassDescriptor("edu/umd/cs/findbugs/annotations/Nullable"),
```

Comment:
Comment it out instead and add a comment line above
// intentionally ignored, semantics is different to other Nullable annotations, see #3243

Reply:
fixed and add comment https://github.com/spotbugs/spotbugs/pull/3288/commits/6db15f5b03d37b186c54b3502f974ff301e375c0